test/functional/board-layout/dnd-page-object.ts

```diff
@@ -174,4 +174,11 @@ export class DndPageObject extends BasePageObject {
     const liveAnnouncements = await this.browser.execute(() => window.__liveAnnouncements ?? []);
     return liveAnnouncements;
   }
+
+  // Clicking on a drag/resize handler performs an animation (show the UAP actions, moving the board item).
```

Comment:
nit: should it be "handle" instead of "handler" (in the comment and the method name)?